.env

```diff
@@ -1,13 +0,0 @@
-# JSON web token (JWT) secret: this keeps our app's user authentication secure
-# This secret should be a random 20-ish character string
-JWT_SECRET ='p4sta.w1th-b0logn3s3-s@uce'
-
-# Mongo DB
-# Local development
-MONGODB_URI='mongodb://localhost/bulletproof-nodejs'
-
-# Port
-PORT=3000
-
-# Debug 
-LOG_LEVEL='debug'
```

Comment:
Why did you remove this file?!

Reply:
Because the best practice on developing with `.env` file is: `.env` file is not supposed to be commited into a repository, as it will accidentally reveal production credentials, as per I have mentioned in my PR note